src/test/java/edu/cornell/kfs/fp/batch/service/impl/CreateAccountingDocumentServiceImplTest.java

```diff
@@ -677,6 +688,10 @@ private void assertCuDisbursementVoucherDocumentsCorrect(CuDisbursementVoucherDo
                 actualDvDocument.getDvNonEmployeeTravel().getDvNonEmployeeExpenses(), this::assertTravelExpenseCorrect);
         assertObjectListIsCorrect("non employee prepaid travel expenses aren't correct", expectedDvDocument.getDvNonEmployeeTravel().getDvPrePaidEmployeeExpenses(), 
                 actualDvDocument.getDvNonEmployeeTravel().getDvPrePaidEmployeeExpenses(), this::assertTravelExpenseCorrect);
+        assertEquals("Due Dates should match", dateFormat.format(expectedDvDocument.getDisbursementVoucherDueDate()), 
+                dateFormat.format(actualDvDocument.getDisbursementVoucherDueDate()) );
+        assertEquals("Invoice Dates should match", actualDvDocument.getInvoiceDate(), expectedDvDocument.getInvoiceDate());
+        assertEquals("Invoice numbers should match", actualDvDocument.getInvoiceDate(), expectedDvDocument.getInvoiceDate());

```

Comment:
These two assertions still need to adjust the positions of the expected-value and actual-value args. Sorry if that wasn't clear from my initial comment.

Reply:
no worries, I think I glossed over them, I fixed